app/models/user.rb

```diff
@@ -24,6 +24,7 @@ class User
 
   field :password_digest
   field :remember_me_cookie
+  attr_accessor: subscribe_newsletter
```

Comment:
Looks like a slight typo here, should be `attr_accessor :subscribe_newsletter`

That said, I think there may be a small bug:

`attr_accessor` means that the field won't be persisted to the database. The first thing `MailChimpWorker` does is load the user's record from the database to check that flag, but it won't be there. 

I think that will mean nobody will be subscribed, is that right?

If so, one option is to either pass the flag into `MailChimpWorker` in the `after_create` method, or only trigger the worker if the flag is set to true. I wouldn't recommend storing the flag in the database, though.